packages/reactivity/src/ref.ts

```diff
@@ -3,11 +3,11 @@ import { OperationTypes } from './operations'
 import { isObject } from '@vue/shared'
 import { reactive } from './reactive'
 
-export const refSymbol = Symbol()
+export const refSymbol = __DEV__ ? Symbol('isRef') : Symbol()
```

Comment:
This somehow breaks tests but we need it for consistency.

Reply:
typescript being typescript...
because it can be 2 symbols I guess. not entirely sure why, but wrote a fix for that